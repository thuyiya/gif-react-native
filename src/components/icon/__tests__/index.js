/* eslint-disable no-undef */
import React from "react";
// import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";
import { Icon } from "_components";

describe("Icon", () => {
  it("renders correctly react-test-renderer", () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Icon name={"search"} />);
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
