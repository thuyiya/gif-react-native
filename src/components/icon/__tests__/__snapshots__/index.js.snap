// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders correctly react-test-renderer 1`] = `
<Image
  source={Object {}}
  style={
    Array [
      Object {
        "height": 24,
        "tintColor": "#333333",
        "width": 24,
      },
      undefined,
    ]
  }
/>
`;
